.github/templates/tilgangsinfo-pnd.md

```diff
@@ -7,24 +7,28 @@ labels: 'kind/deploy_patch, team/tilgangsinfo'
 - [ ] Deploy[^1] all components to production[^2].
   - [ ] Deploy [Authentication] to production.
   - [ ] Deploy [Resource Registry] to production.
+  - [ ] Deploy [^4] [Auditlog](https://github.com/Altinn/altinn-auth-audit-log/actions) to production.
```

Comment:
This link is still not pointing to the correct action. And it should be in the same format as the other links given that it's repeated in multiple places in this document.

Reply:
The actual action is given in the footnote. So i think there is no need for a link here

Reply:
You have a link here though...